scrapers/CrashpadSeries.yml

```
@@ -0,0 +1,33 @@
+name: CrashpadSeries 
+debug:
+  printHTML: true
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://crashpadseries.com
```

If there is not specific reason we usually do it like this
```suggestion
      - crashpadseries.com
```

---

fixed